lib/livebook/session/data.ex

```diff
@@ -55,7 +55,9 @@ defmodule Livebook.Session.Data do
           revision: cell_revision(),
           deltas: list(Delta.t()),
           revision_by_client_pid: %{pid() => cell_revision()},
-          evaluation_digest: String.t() | nil
+          evaluation_digest: String.t() | nil,
+          evaluation_start_time: float(),
+          last_evaluation_time: float()
```

Comment:
It's helpful to keep unit in the name, like `evaluation_time_ms`. Also, it should probably be `integer() | nil` :)